q2_diversity/tests/test_beta_correlation.py

```diff
@@ -0,0 +1,87 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2018, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# TODO Cleanup dependencies
+# import io
+import unittest
+
+import tempfile
+# import biom
+import skbio
+# import numpy as np
+import pandas as pd
+# import pandas.util.testing as pdt
+import qiime2
+from qiime2 import Artifact
+
+from qiime2.plugin.testing import TestPluginBase
+# from qiime2 import Artifact, Metadata
+# from q2_diversity._beta._beta_correlation import beta_correlation
+
+
+class BetaCorrelationTests(TestPluginBase):
```

Comment:
Tests look good - I asked a question above about ID subsets --- that will most likely need a test included here.

Reply:
This test was nuked along with the code block it was testing, as discussed above.